content-blocker-lib-ios/ContentBlockerGenerator/Sources/ContentBlockerGenerator/FileCategory.swift

```diff
@@ -0,0 +1,43 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import Foundation
+
+/// Those are the raw files taken as input for this script
+enum FileCategory: String {
+    case advertising
+    case analytics
+    case content
+    case cryptomining
+    case entity
+    case fingerprinting
+    case social
+
+    func getPath(inputDirectory: String) -> String {
+        return "\(inputDirectory)/\(location)"
+    }
+
+    private var location: String {
+        switch self {
+        case .advertising:
+            return "normalized-lists/ads-track-digest256.json"
+        case .analytics:
+            return "normalized-lists/analytics-track-digest256.json"
+        case .content:
+            return "normalized-lists/content-track-digest256.json"
+        case .cryptomining:
+            return "normalized-lists/base-cryptomining-track-digest256.json"
+        case .entity:
+            return "disconnect-entitylist.json"
+        case .fingerprinting:
+            return "normalized-lists/base-fingerprinting-track-digest256.json"
+        case .social:
+            return "normalized-lists/social-track-digest256.json"
+        }
+    }
```

Comment:
Does this capture all of the ones below:

<img width="607" alt="image" src="https://user-images.githubusercontent.com/8919439/208769573-c637e588-cefa-49ce-9a80-7faf12ceff6d.png">

Reply:
Not all the lists no. This will be optimized with Isabel's changes too! I just picked the whole folder out of simplicity